api-objects/UsersObject.ts

```diff
@@ -1,10 +1,9 @@
 export enum AuthorityEnum { 
 	Audiologist, 
-	Statistician, 
 	Admin,
 }
 
-export const authorityTypes: Array<string> = ['Audiologist', 'Statistician', 'Admin'];
+export const authorityTypes: Array<string> = ['Audiologist', 'Admin'];
```

Comment:
The Database needs to be updated as-well, this will confuse the admin login unless the ENUM is also changed.